src/index.d.ts

```diff
@@ -1,11 +1,11 @@
+import type { Config } from "@jest/types";
```

Comment:
would `@jest/types` need to be installed as a dependency for this project in order for this to work?

Reply:
Hmmm, intereresting.

`@jest/types` is a [dependancy of `jest` core](https://github.com/jestjs/jest/blob/main/packages/jest/package.json).

It's **highly** likely that anyone using this will also have jest installed but it's not explicitly a dependancy of `jest-a-coverage-slip-detector`. I sort of feel like what might make the most sense would be to add `jest` as a `peerDependencies` here. What do you think of that direction?

Reply:
fair point - and yeah, a peer dependency makes sense to me, makes it more explicit